Gruntfile.js

```diff
@@ -1819,7 +1818,7 @@ module.exports = function(grunt) {
 			if ( action !== 'deleted' ) {
 				grunt.config( [ 'copy', 'dynamic-js', 'files' ], files );
 			}
-		// For the webpack builds configure the jsvalidate task to only check those files build by webpack.
+		// For the webpack builds configure the task to only check those files built by webpack.
```

Comment:
I'm curious now that we're removing jsvalidate if this whole `else if` could be removed or if it's still useful for some reason. I don't see that as a blocker to this patch though, so going to pre-approve.

Reply:
I wasn't certain so erred on the side of not changing the block, since it affects a non-validate-specific task.